app/helpers/question_view_helper.rb

```diff
@@ -22,6 +22,6 @@ def label_size(page_header, conditional)
   def label_tag(page_header, conditional)
     return "" if conditional
 
-    page_header.blank? ? "h1" : "h2"
+    page_header.blank? ? "h1" : "div"
```

Comment:
Given this is passed to the form builder, I wonder if we can provide `nil`?

Reply:
yep that appears to render `<div>` also

Reply:
In which case, probably best to leave is as `div`, as that’s more explicit. 👍

Reply:
(I thought `nil` would remove the wrapping div, but I guess that’s still required because `legend` be `legend`)